src/dualscreen.tsx

```diff
@@ -445,17 +446,20 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
       this._switchToHidden();
       return;
     }
-    if (slideItem.portrait !== this._pipPortraitMode) {
-      this._pipPortraitMode = slideItem.portrait;
-      if (this._layout === Layout.PIP) {
-        // update PIP component
-        this._setMode();
-        return;
-      }
-    }
-    // apply dual-screen layout
+
+    let originalHiddenLayout = false;
```

Comment:
I think here we created extra variable
on line 461 we can compare:
```
if (originalHiddenLayout || (slideItem.portrait !== this._pipPortraitMode && this._layout === Layout.PIP)) {
```
and totally remove if block from line 456 to. 459